lib/models/os.ts

```diff
@@ -49,6 +49,11 @@ const getOsModel = function (
 	const applicationModel = once(() =>
 		require('./application').default(deps, opts),
 	);
+	const deviceTypesUtils = once(
```

Comment:
If you check the cli lazy.ts pattern I have all the lazy load imports set up in one file so it can easily be shared by everything and only get typed once, you could also combo it with the import blacklist to enforce that only the lazy file can actually import them

Reply:
Oh nice :+1: 
Is it fine if I leave that for a follow-up?